cpp/memilio/io/result_io.h

```diff
@@ -121,6 +121,29 @@ IOResult<void> save_result_with_params(const std::vector<TimeSeries<double>>& re
     return success();
 }
 
+/**
+ * @brief Save the results of the edges for a single graph simulation run.
+ * @param result Simulation results per edge of the graph.
+ * @param ids Identifiers for the start and end node of the edges.
+ * @param filename Name of file
+ * @return Any io errors that occur during writing of the files. 
+ */
+IOResult<void> save_edges(const std::vector<TimeSeries<double>>& results, const std::vector<std::pair<int, int>>& ids,
+                          const std::string& filename);
+
+/**
+ * Save the results for the Edges obtained from the function condense_m_mobility.
```

Comment:
You cannot use the name of a private member of an unnamed class in the documentation here. No one will be able to understand this connection